store/blocks/header/header.jsonc

```diff
@@ -7,21 +7,21 @@
   },
   "header-layout.desktop": {
     "children": [
-      "header-row#1-desktop",
-      "header-row#2-desktop",
-      "header-row#3-desktop",
-      "header-row#4-desktop"
+      "flex-layout.row#1-desktop",
+      "flex-layout.row#2-desktop",
+      "flex-layout.row#3-desktop",
+      "sticky-layout.stack-container#4-desktop"
```

Comment:
Why is that needed? I think it could be if you use more than one sticky-layout right? But in this case it's not needed, right?

Reply:
Right, it's not needed, but I tried keeping the same functionalities the previous header had. Do you prefer if I remove it?